src/components/CandidateContest.tsx

```diff
@@ -526,7 +526,7 @@ class CandidateContest extends React.Component<Props, State> {
                   {contest.allowWriteIns && !hasReachedMaxSelections && (
                     <Choice
                       isSelected={false}
-                      onPointerUp={this.initWriteInCandidate}
+                      onClick={this.initWriteInCandidate}
```

Comment:
so does this make this action less sensitive?

Reply:
Yes, hence my desire to convert/simplify these to button elements. Glad you asked/reminded me of this reason.